src/client/app/left/date-picker/date-picker.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="row date-picker">
+  <br/>
+  <div class="col-md-3">Please select the date: </div>
+  <div class="col-md-9">
+    <input type="date" min="2011-01-01" max={{max}} value="2017-02-27" (change)="datePickerUpdated($event)"/>
```

Comment:
yeah, safari doesn't support it... can we do the demo on Chrome? I'm guessing a huge amount of time would be spent on integrating JQuery into karma and jasmine, but we haven't finished core component development yet :)